packages/gafl-webapp-service/src/processors/licence-type-display.js

```diff
@@ -28,11 +28,11 @@ export const licenceTypeDisplay = (permission, mssgs) => {
 export const licenceTypeAndLengthDisplay = (permission, mssgs) => {
   switch (permission.licenceLength) {
     case '12M':
-      return `${licenceTypeDisplay(permission, mssgs)}, 12 months`
+      return `${licenceTypeDisplay(permission, mssgs)}, ${mssgs.licence_type_12m}`
     case '8D':
-      return `${licenceTypeDisplay(permission, mssgs)}, 8 days`
+      return `${licenceTypeDisplay(permission, mssgs)}, ${mssgs.licence_type_8d}`
     default:
-      return `${licenceTypeDisplay(permission, mssgs)}, 1 day`
+      return `${licenceTypeDisplay(permission, mssgs)}, ${mssgs.licence_type_1d}`
```

Comment:
Perhaps beyond the scope of this ticket, but could you rewrite this to eliminate the case statement and return in a single line?